Assets/Scripts/EOSAchievementManager.cs

```diff
@@ -20,100 +20,108 @@
+    /// Class <c>EOSAchievementManager</c> is a simplified wrapper for
+    /// EOS [Achievements Interface](https://dev.epicgames.com/docs/services/en-US/Interfaces/Achievements/index.html).
     /// </summary>
-
     public class EOSAchievementManager : IEOSSubManager, IEOSOnConnectLogin
     {
-        private ConcurrentDictionary<string, byte[]> downloadCache;
-        private List<DefinitionV2> achievementDefinitionCache;
-        //private ConcurrentDictionary<ProductUserId, UserInfoData> productUserIdToUserInfoData;
-        private ConcurrentDictionary<ProductUserId, List<PlayerAchievement>> productUserIdToPlayerAchievement;
-        private AchievementsInterface eosAchievementInterface;
-
-        private Dictionary<ProductUserId, List<Stat>> productUserIdToStatsCache = new Dictionary<ProductUserId, List<Stat>>();
-
-        private List<Action> achievementDataUpdatedCallbacks;
+        /// <summary>
+        /// Stores data that has been cached, where the string key is a URI,
+        /// and the value is the bytes of the resource at that URI.
+        /// </summary>
+        private static ConcurrentDictionary<string, byte[]> _downloadCache = new();
 
-        public EOSAchievementManager()
-        {
-            downloadCache = new ConcurrentDictionary<string, byte[]>();
-            achievementDefinitionCache = new List<DefinitionV2>();
-            //productUserIdToUserInfoData = new ConcurrentDictionary<ProductUserId, UserInfoData>();
-            productUserIdToPlayerAchievement = new ConcurrentDictionary<ProductUserId, List<PlayerAchievement>>();
-            achievementDataUpdatedCallbacks = new List<Action>();
-        }
+        /// <summary>
+        /// Contains a list of the achievements that exist for the game.
+        /// </summary>
+        private IList<DefinitionV2> _achievements = new List<DefinitionV2>();
+        
+        /// <summary>
+        /// Maps a given user to a list of player achievements.
+        /// </summary>
+        private ConcurrentDictionary<ProductUserId, List<PlayerAchievement>> _playerAchievements = new();
+        
+        /// <summary>
+        /// Maps a given user to a list of player statistics.
+        /// </summary>
+        private IDictionary<ProductUserId, List<Stat>> _playerStats = new Dictionary<ProductUserId, List<Stat>>();
 
+        /// <summary>
+        /// Stores a list of functions to be called whenever data related to
+        /// Achievements has been updated locally.
+        /// </summary>
+        private IList<Action> _dataUpdatedNotifiers = new List<Action>();
 
+        /// <summary>
+        /// Conditionally executed proxy function for Unity's log function.
+        /// </summary>
+        /// <param name="toPrint">The message to log.</param>
         [Conditional("ENABLE_DEBUG_EOSACHIEVEMENTMANAGER")]
-        static void print(string toPrint)
+        private static void Log(string toPrint)
```

Comment:
Should we change this from print to `Log`? What was the original intent behind the `print` signature?